llvm-runtimes/compiler-rt-sanitizers/compiler-rt-sanitizers-15.0.7-r1.ebuild

```diff
@@ -96,6 +96,8 @@ src_prepare() {
 	# bug #926330
 	sed -i -e '/-Wthread-safety/d' CMakeLists.txt cmake/config-ix.cmake || die
 
+	sed -e '/^#include <cassert>$/a#include <cstdint>' -i lib/orc/error.h || die
```

Comment:
Meh. We have patchset for a reason. Can you find upstream commit, ideally?

Reply:
I didn't understand a word you just said. It's just a missing include (probably exposed by GCC 15 transitively including fewer headers). If you want to stuff this somewhere else, have at it.